services/report/raw_upload_processor.py

```diff
@@ -22,23 +22,9 @@
 
 log = logging.getLogger(__name__)
 
-GLOBAL_LEVEL_LABEL = (
-    SpecialLabelsEnum.CODECOV_ALL_LABELS_PLACEHOLDER.corresponding_label
-)
-
-
-# This is a lambda function to return different objects
-def DEFAULT_LABEL_INDEX():
-    return {
-        SpecialLabelsEnum.CODECOV_ALL_LABELS_PLACEHOLDER.corresponding_index: SpecialLabelsEnum.CODECOV_ALL_LABELS_PLACEHOLDER.corresponding_label
-    }
-
-
-def invert_pattern(string: str) -> str:
-    if string.startswith("!"):
-        return string[1:]
-    else:
-        return "!%s" % string
+DEFAULT_LABEL_INDEX = {
```

Comment:
Is it fine to use a shared reference as the label index?

Reply:
actually you are right, I fixed this to default the `labels_index` to a copy of that global.
I might have been confused by the comment, plus this being used for a comparison later on which in python is structural equality for dicts.